var/spack/repos/builtin/packages/moab/package.py

```diff
@@ -58,6 +58,7 @@ class Moab(Package):
     depends_on('mpi')
     depends_on('hdf5+mpi')
     depends_on('netcdf+mpi', when='+netcdf')
```

Comment:
Hmm. More I think about this...I conclude it would need a plain `depends_on(netcdf)` _only_ if building with and without MPI is supported. MPI dependence is UNconditiional here. Maybe I should make at least that variant here?